src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -2052,39 +2052,46 @@
 
     /**
-     * Represents the {@link Key} for the "scale" for the size of an
-     * {@link Entity}.
+     * The "scale" for the size of an {@link Entity}.
+     *
+     * <p>Together with {@link #BASE_SIZE} and {@link #HEIGHT} this defines the size of an {@link Entity}.</p>
      */
     public static final Supplier<Key<BoundedValue<Double>>> SCALE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SCALE");
 
     /**
-     * Represents the {@link Key} for a {@link Fox fox's} second trusted {@link UUID}, usually a {@link Player}.
+     * A {@link Beacon}'s secondary effect.
      */
-    public static final Supplier<Key<Value<UUID>>> SECOND_TRUSTED = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SECOND_TRUSTED");
+    public static final Supplier<Key<Value<PotionEffectType>>> SECONDARY_EFFECT = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SECONDARY_EFFECT");
 
     /**
-     * Represents the {@link Key} for the shooter of a {@link Projectile}.
+     * A {@link Fox fox's} second trusted {@link UUID}, usually a {@link Player}.
      */
-    public static final Supplier<Key<Value<ProjectileSource>>> SHOOTER = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SHOOTER");
+    public static final Supplier<Key<Value<UUID>>> SECOND_TRUSTED = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SECOND_TRUSTED");
 
     /**
-     * Represents the {@link Key} for representing the "should drop" state
-     * of a {@link BlockState}.
+     * The shooter of a {@link Projectile}.
      */
-    public static final Supplier<Key<Value<Boolean>>> SHOULD_DROP = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SHOULD_DROP");
+    public static final Supplier<Key<Value<ProjectileSource>>> SHOOTER = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SHOOTER");
 
     /**
-     * Represents the {@link Key} for if a {@link EnderCrystal} should show it's bottom bedrock platform.
+     * Whether a {@link EnderCrystal} should show it's bottom bedrock platform.
      */
     public static final Supplier<Key<Value<Boolean>>> SHOW_BOTTOM = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SHOW_BOTTOM");
 
     /**
-     * Represents the {@link Key} for the lines displayed on a {@link Sign}.
+     * The lines displayed on a {@link Sign}.
      */
     public static final Supplier<Key<ListValue<Text>>> SIGN_LINES = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SIGN_LINES");
 
     /**
-     * Represents the {@link Key} for the skin of a {@link Humanoid}.
+     * The size of a {@link Slime}.
+     * or
+     * The size of a {@link Phantom}. In vanilla, this ranges between 0 and 64.
+     */
+    public static final Supplier<Key<BoundedValue<Integer>>> SIZE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "SIZE");
```

Comment:
I wouldn't name it `SIZE`, since it will likely conflict with possible future keys. So maybe just `SLIME_SIZE` and `PHANTOM_SIZE`, but I'm not sure

Reply:
yeah I'm not sure either